### PR TITLE
homework submission week1

### PR DESCRIPTION
**Pull request by hnnhack**

### CODE REVIEW

File: homework-submission/hw-week1/src/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  itemDone() {
+    this.setState({ done: !this.state.done });
+  }
+
+  render() {
+    const { description, deadline } = this.props;
+
+    return (
+      <li
+        onClick={this.itemDone}
```
Comment (ArcoMul) [inline homework-submission/hw-week1/src/index.js]: Cool that you already implemented a click event!

File: homework-submission/hw-week1/src/index.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const staticList = [
```
Comment (ArcoMul) [inline homework-submission/hw-week1/src/index.js]: As far as I can see you are not using this `staticList` variable. The static list is hardcoded so I think you can remove this?

File: homework-submission/hw-week1/src/index.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  {
+    id: 3,
+    description: 'Eat breakfast',
+    deadline: '2017-09-09',
+    done: false,
+  },
+];
+
+// This component is header aimed for top of each list
+const Header = ({ name }) => <h1 className="header">{name}</h1>;
```
Comment (ArcoMul) [inline homework-submission/hw-week1/src/index.js]: Nice function component!

File: homework-submission/hw-week1/src/index.js (hunk 2 above)
Comment (hnnhack) [reply homework-submission/hw-week1/src/index.js]: Actually, I used it in staticComponent in my file line 71.

**Comment by hnnhack** [comment]
Hi @ArcoMul! 
Thank you very much for your kind reviews.

**Pull request closed without merging**